openapi.yml

```diff
@@ -704,6 +710,12 @@ components:
         request_time:
           type: string
           format: date-time
+        max_paths:
+          type: integer
+          format: int32
+          description: Maximum number of paths to be considered. Default value is 2.
```

Comment:
You can set `default: 2`, similar to what we have on other default int(s), and remove the default comment from the description.

Reply:
Added.